lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Task/TaskRuntime.cs

```diff
@@ -174,6 +174,21 @@ public void Close(byte[] message)
                 _currentStatus.SetException(TaskClientCodeException.Create(
                     TaskId, ContextId, "Error during Close().", e));
             }
+            finally
+            {
+                try
+                {
+                    if (_userTask != null)
+                    {
+                        _userTask.Dispose();
+                    }
+                }
+                catch (Exception e)
+                {
+                    Utilities.Diagnostics.Exceptions.Caught(
```

Comment:
No, and that would be correct, wouldn't it be? If we reach this state, the task's `Dispose` method threw an exception. There is no clean way to recover from that, as far as I know.